pennylane/ops/qubit.py

```diff
@@ -1012,6 +1033,10 @@ def decomposition(phi, wires):
     def adjoint(self):
         return ControlledPhaseShift(-self.data[0], wires=self.wires)
 
+    @property
+    def comp_control_wires(self):
+        return Wires(self.wires[0])
+
```

Comment:
Not sure how to handle testing for this one, it seems redundant to do it for every gate.

Reply:
I've added the `# pragma` (L595 of `operation.py`)  but the warning still occurs. Was it added incorrectly?

Reply:
Looks okay to me 🤔

Reply:
Huh :( Well... since it is not causing any issues with the CI and doesn't require an override, I am going to just merge :+1: